wiki/Medals/Unlock_requirements/Dedication/de.md

```diff
@@ -40,3 +35,11 @@ outdated_since: 4c74ad9e2a804492b30c360210020a682b4937f8
 | 400.000 Keys | Treffe insgesamt 400.000 Tasten in osu!mania. | Four hundred thousand and still not even close. |
 | 4.000.000 Keys | Treffe insgesamt 4.000.000 Tasten in osu!mania. | Is this the end of the rainbow? |
 | 40.000.000 Keys | Treffe insgesamt 40.000.000 Tasten in osu!mania. | When someone asks which keys you play, the answer is now 'yes'. |
+
+## Tägliche Herausforderung
+
+| Medaillenname | Anforderung | Beschreibung |
+| :-- | :-- | :-- |
+| Daily Sprout | Erreiche eine 1-tägige Serie in [Tägliche Herausforderung](/wiki/Gameplay/Daily_challenge). | Ready for anything. |
```

Comment:
I don't think I've heard a streak be translated as "Serie" before.. Is there a precedent for this? I would've imagined the translation would need to be less literal by doing something akin to `Gewinne/Bestehe die [taegliche Herausforderung](...) sieben Mal hintereinander`

Reply:
This sounds even better 👍 

```suggestion
| Daily Sprout | Gewinne die [tägliche Herausforderung](/wiki/Gameplay/Daily_challenge) einmal. | Ready for anything. |
```